recipes/libbacktrace/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+./configure --prefix="${PREFIX}"
+make
```

Comment:
Is there a `make check`?

Reply:
Yes, just added it. I also noticed that it links to zlib, so I added it as an explicit build/run dependency.